segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/tables/TableStore.java

```diff
@@ -130,6 +130,35 @@
      */
     CompletableFuture<Void> deleteSegment(String segmentName, boolean mustBeEmpty, Duration timeout);
 
+    /**
```

Comment:
These methods have been removed in 0.10. Why are you bringing them back?

Reply:
This was an error introduced when merging from upstream branch. Fixed now.